packages/key-management/test/util/mapHardwareSigningData.test.ts

```diff
@@ -0,0 +1,25 @@
+import { CML } from '@cardano-sdk/core';
+import { ManagedFreeableScope } from '@cardano-sdk/util';
+import { mintValueAsString } from '../../src/util';
+
+describe('KeyManagement.util.mapHardwareSigningData', () => {
+  let scope: ManagedFreeableScope;
+
+  beforeEach(() => {
+    scope = new ManagedFreeableScope();
+  });
+  afterEach(() => {
+    scope.dispose();
+  });
+  describe('mintValueAsString', () => {
+    it('returns "1" when passed a CML.Int representing 1', () => {
+      expect(mintValueAsString(scope, scope.manage(CML.Int.from_str('1')))).toBe('1');
+    });
+    it('returns "-1" when passed a CML.Int representing -1', () => {
+      expect(mintValueAsString(scope, scope.manage(CML.Int.from_str('-1')))).toBe('-1');
+    });
+    it('returns "0" when passed a CML.Int representing 0', () => {
+      expect(mintValueAsString(scope, scope.manage(CML.Int.from_str('0')))).toBe('0');
+    });
+  });
```

Comment:
I think a test capturing [the limits cases](https://github.com/dcSpark/cardano-multiplatform-lib/blob/685354495562ff879101fd023eb89555db9bbb1e/rust/pkg/cardano_multiplatform_lib.js.flow#L3369) would be helpful.
Although we would just be testing the CML implementation, so maybe we shouldn't add it 😄

Reply:
Removed in https://github.com/input-output-hk/cardano-js-sdk/commit/67e3f4aa61259851b735f3652c1fd95e64000848